File: client/python/get.py
import sys
import grpc

sys.path.append('../../../microservices-proto/python/order')

import order_pb2
import order_pb2_grpc

def run():
    with grpc.insecure_channel('127.0.0.1:3000') as channel:
        client = order_pb2_grpc.OrderStub(channel)

        request = order_pb2.GetOrderRequest(
            order_id=3
        )

        response = client.Get(request)

        print("Order created:", response)

if __name__ == '__main__':
    run()

File: client/golang/get.go
package main

import (
	"context"
	"log"
	"time"

	"github.com/k0ntra/microservices-proto/golang/order"
	"google.golang.org/grpc"
)

func main() {
	conn, err := grpc.Dial("127.0.0.1:3000", grpc.WithInsecure())
	if err != nil {
		log.Fatalf("Did not connect: %v", err)
	}
	defer conn.Close()

	client := order.NewOrderClient(conn)

	req := &order.GetOrderRequest{
		OrderId: 1,
	}

	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
	defer cancel()

	resp, err := client.Get(ctx, req)
	if err != nil {
		log.Fatalf("Could not get order: %v", err)
	}

	log.Printf("Order: %v", resp)
}
